silkrpc/consensus/ethash.cpp

```diff
@@ -16,15 +16,43 @@
 
 #include "ethash.hpp"
 
+#include <silkworm/core/silkworm/chain/protocol_param.hpp>
+#include <silkrpc/json/types.hpp>
+
 namespace silkrpc::ethash {
 
 BlockReward compute_reward(const ChainConfig& config, const silkworm::Block& block) {
-    return BlockReward{};
+    auto chain_config = silkworm::ChainConfig::from_json(config.config).value();
+    auto revision = chain_config.revision(block.header.number);
```

Comment:
`revision` should be declared `const`, right?

Reply:
Right.